### ISSUE TITLE
[a11y tool] inform users that the tool is available

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#25]
Since some combinations of AT+browsers do not announce `role=math`, can the a11y tool add a label to inform users of the tool's presence? Perhaps `role=application` is better supported (if that's suitable)?

**Comment by zorkow** [comment]
See https://github.com/mathjax/MathJax-RespEq/pull/48
